inc/class-registry.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Component registry.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving\Components;
+
+/**
```

Comment:
The `class-registry.php` file should only define the class. Any additional helper functions should go in a `registry.php` file or a generic `inc/components/functions.php` file.